Getting io/dekorate/helm/config/HelmChartConfigFluentImpl error using Quarkus 3.5.0

It might be early, but building with the Quarkus 3.5.0 release (because Renovate updated it) gives this error that I comes from the Helm extension:

```
[ERROR]         [error]: Build step io.quarkiverse.helm.deployment.HelmProcessor#generateResources threw an exception: java.lang.NoClassDefFoundError: io/dekorate/helm/config/HelmChartConfigFluentImpl
[ERROR]         at io.quarkiverse.helm.deployment.HelmProcessor.toDekorateHelmChartConfig(HelmProcessor.java:375)
[ERROR]         at io.quarkiverse.helm.deployment.HelmProcessor.doGenerateResources(HelmProcessor.java:188)
[ERROR]         at io.quarkiverse.helm.deployment.HelmProcessor.generateResources(HelmProcessor.java:160)
[ERROR]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[ERROR]         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR]         at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[ERROR]         at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:849)
[ERROR]         at io.quarkus.builder.BuildContext.run(BuildContext.java:282)
[ERROR]         at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR]         at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
[ERROR]         at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
[ERROR]         at java.base/java.lang.Thread.run(Thread.java:833)
```

Here is a small reproducer created from the Quarkus app generator (v3.5.x) with the following extensions:

```xml
<dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-keycloak-authorization</artifactId>
    </dependency>
    <dependency>
      <groupId>io.quarkiverse.helm</groupId>
      <artifactId>quarkus-helm</artifactId>
      <version>1.1.0</version>
    </dependency>
    <dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-keycloak-admin-client-reactive</artifactId>
    </dependency>
    <dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-smallrye-stork</artifactId>
    </dependency>
```

Run `mvn install -U` gives a similar exception:

`[error]: Build step io.quarkiverse.helm.deployment.HelmProcessor#generateResources threw an exception: java.lang.NoClassDefFoundError: io/dekorate/helm/config/HelmChartConfigFluentImpl`

```
[ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.5.0:build (default) on project code-with-quarkus: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR]         [error]: Build step io.quarkiverse.helm.deployment.HelmProcessor#generateResources threw an exception: java.lang.NoClassDefFoundError: io/dekorate/helm/config/HelmChartConfigFluentImpl
[ERROR]         at io.quarkiverse.helm.deployment.HelmProcessor.toDekorateHelmChartConfig(HelmProcessor.java:375)
[ERROR]         at io.quarkiverse.helm.deployment.HelmProcessor.doGenerateResources(HelmProcessor.java:188)
[ERROR]         at io.quarkiverse.helm.deployment.HelmProcessor.generateResources(HelmProcessor.java:160)
[ERROR]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[ERROR]         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR]         at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[ERROR]         at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:849)
[ERROR]         at io.quarkus.builder.BuildContext.run(BuildContext.java:282)
[ERROR]         at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR]         at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
[ERROR]         at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
[ERROR]         at java.base/java.lang.Thread.run(Thread.java:833)
[ERROR]         at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] Caused by: java.lang.ClassNotFoundException: io.dekorate.helm.config.HelmChartConfigFluentImpl
[ERROR]         at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
[ERROR]         at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
[ERROR]         at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
[ERROR]         at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
[ERROR]         at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:516)
[ERROR]         at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:466)
[ERROR]         ... 14 more
```


[code-with-quarkus.zip](https://github.com/quarkiverse/quarkus-helm/files/13205062/code-with-quarkus.zip)

--- a/.github/project.yml
+++ b/.github/project.yml
@@ -1,3 +1,3 @@
 release:
-  current-version: 1.1.0
-  next-version: 1.1.1-SNAPSHOT
+  current-version: 1.2.0
+  next-version: 1.2.1-SNAPSHOT

